R/ui_page_heading.R

```diff
@@ -6,7 +6,7 @@
 #'
 #' @return a shiny tag
 #'
-page_heading <- function(pretitle = "Page pretitle", title = "Page title", ...){
+page_heading <- function(pretitle = "Page pretitle", title = "Page title", download_text = "Download text", ...){
```

Comment:
The button looks great! I would modify the function a bit to make it more flexible so that different pages can have different buttons (or no buttons at all). Instead of having the text of the button as an argument we could have a separate function that creates a "download report button". That function will contain the code you wrote in this function so that a page heading with a button could be created with something like `page_heading(download_report_button(text = "Download report"))` this will work because it will put the button where the dots (...) are in the function. A page heading with no button can also be easily created and it would even work if we want something completely different to a button in the future.